christfellowshipchurch/web-app-v2
# Changed Boynton Beach translations time and YA typo for all locations

### About
Changed Boynton Beach translations time and YA typo for all locations

### Test Instructions
Open Locations -> Boynton Beach, the translations services should now say 10am
Under YA it should say "The perfect place **for** those who..." (added the word "for" to fix the typo)

### Closes Tickets
[CFDP-2278](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2278)

### Related Tickets
N/A
